### ISSUE TITLE
[LVGL] Chart has no personal settings

### ISSUE THREAD

**Issue by elmonic** [eez-open/studio#482]
Hi
It is better to add settings like below to the chart.
If its parameters can be adjusted so that we have a live display, it will be very good.
Change Setting:

![setting1](https://github.com/user-attachments/assets/52e2e1a2-f769-48e7-a1ab-eafc4bbde9fa)

Change X , Y axis Setting

![setting 2](https://github.com/user-attachments/assets/f859b120-a658-49bf-a4e3-e7d158f52e0c)

add Data to chart

![setting 3](https://github.com/user-attachments/assets/7ee0db41-a1a7-4557-87c8-7e62a2925b0a)

**Comment by elmonic** [comment]
hi
Will it be added in the next version?

**Comment by mvladic** [comment]
Yes, there is a good chance that this will be done in the next release.

**Comment by littleboot** [comment]
I was/am also looking for this functionality, I switched from squareline studio and was trying out everything in EEZ Studio I thought the chart widget was broken until I found this issue report (Its just not fully implemented yet, all I wanted to know).

**Comment by dpeart** [comment]
How can we do this today as we wait for the enhancement?

I assume I need to do this with a call back of my own.

thanks,

**Comment by mvladic** [comment]
> How can we do this today as we wait for the enhancement? 

Create a Chart widget with some name (let say `my_chart`) and then after `ui_init()` is called, do the remaining initialization by using `objects.my_chart` as widget pointer.
